src/19 - Summaries.rsc

```diff
@@ -1293,6 +1293,7 @@ Macro "VMT_Delay Summary" (Args)
   RunMacro("Close All")
 
   // Summarize by different variable
+  group_fields = group_fields + {{"County", "HCMType"}} //add VMT by facility type by county
```

Comment:
group_fields already includes both HCMType and County right? See line 1194

Reply:
Yes, that creates VMT_Delay_by_HCMType.csv and VMT_Delay_by_County.csv. CAMPO wants VMT_Delay_by_HCMType_by_County, which is grouped by two variables.